lib/websocket/websocket-handler.ts

```diff
@@ -152,7 +158,7 @@ function setup(socket: Interface.ISocket): void
                         activeSubscriptions.set(s.correlationId, s);
                     });
                     socket.log.debug('Subscribed');
-                    socket.notifySubscribed();
+                    socket.notifySubscribed(getCorrelationIds(subscriptions));
```

Comment:
I don't have a strong feeling of this, but since we have the loop of all `subscriptions` @line 157, it might be advantageous to build the array there for performance reason. Same for unsubscribe below.